native/sapp-linux/src/lib.rs

```diff
@@ -473,8 +474,14 @@ pub unsafe extern "C" fn _sapp_x11_create_window(mut visual: *mut Visual, mut de
         1 as libc::c_int,
     );
     let mut hints = XAllocSizeHints();
-    (*hints).flags |= PWinGravity;
+    (*hints).flags |= PWinGravity | PMinSize | PMaxSize;
```

Comment:
Should `PMinSize | PMaxSize` be used only when `window_resizable == false`?

Reply:
Yes you are right. Let me update this